liblastlog2/meson.build

```diff
@@ -50,9 +48,9 @@ if build_liblastlog2
     exe = executable(
       test_name,
       'src/tests/tst_' + lastlog2_test + '.c',
-      include_directories : [dir_include, dir_liblastlog2],
-      link_with : [lib_common, lib_lastlog2],
-      dependencies : libdl
+      include_directories : [dir_include],
```

Comment:
Not sure if this is correct. We need to install the header file into "liblastlog2" subdirectory. For other libs, we use, for example:
```
include_directories : [dir_include, dir_libblkid]
```
The usual final path should be _/usr/inlude/liblastlog2/lastlog2.h_ and in lastlog2.pc need to use `Cflags: -I${includedir}/liblastlog2`

Reply:
The new dependency below carries both the library and the same include dir.